src/main/java/br/com/tech/challenge/domain/dto/ClienteDTO.java

```diff
@@ -0,0 +1,23 @@
+package br.com.tech.challenge.domain.dto;
+
+import br.com.tech.challenge.domain.entidades.Pedido;
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class ClienteDTO {
+
+    private Long id;
+
+    private String nome;
+
+    private String cpf;
+
+    private String email;
+
+    private Pedido pedido;
```

Comment:
Aqui acho que não precisamos referenciar o pedido, visto que a API pode ser utilizada no início do fluxo, quando o pedido ainda não foi feito.

Reply:
Concordo